### PR TITLE
Extend backend logs API for container

### PR DESCRIPTION
**Pull request by zreigz**
- Add Unit Tests

### CODE REVIEW

**Comment by zreigz** [comment]
PTAL

File: src/app/backend/logs.go [hunk 1]
```diff
@@ -27,13 +27,14 @@ type Logs struct {
 	// Pod id
 	PodId string `json:"podId"`
 	// Specific date (RFC3339) when logs started
-	SinceTime string `json:"sinceTime"`
+	SinceTime unversioned.Time `json:"sinceTime"`
 	// Logs string
 	Logs []string `json:"logs"`
 }
 
 // Return logs for particular pod or error when occurred.
```
Comment (bryk) [inline src/app/backend/logs.go]: Update the docstring to reflect the new param.

**Comment by bryk** [comment]
Looks good. Please rebase your branch. It has conflicts.

File: src/app/backend/logs.go [hunk 2]
```diff
@@ -27,13 +27,14 @@ type Logs struct {
 	// Pod id
 	PodId string `json:"podId"`
 	// Specific date (RFC3339) when logs started
```
Comment (bryk) [inline src/app/backend/logs.go]: Update the docstring. It is not RFC... anymore.

Comment (zreigz) [inline src/app/backend/logs.go]: Done

File: src/app/backend/logs.go (hunk 1 above)
Comment (zreigz) [inline src/app/backend/logs.go]: Done

**Comment by zreigz** [comment]
PTAL

**Pull request merged**